product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md

```diff
@@ -25,6 +27,11 @@ Das _fiskaltrust.Middleware.SCU.DE.Epson_-Package ermöglicht die Verbindung der
 - **Port:** Port des Epson Servers, Printers oder lokjalen Webservices, üblicherweise 8009
 - **DeviceId:** Diese Einstellung ist nur für den Epson Server relevant und dient der Identifikation des Servers. Die DeviceId kann im Epson Server Dashboard gefunden werden, welches über die Host IP Adresse aufgerufen werden kann.
 
+## Troubleshooting:
+
+- Wenn die Developer-TSE einen Selbsttest verlangt obwohl sie aus- und eingesteckt wurde, führen Sie bitte an der Developer-TSE einen Factory-Reset gemäß den Hersteller-Angaben durch.
+- Bei der Verwendung einer Epson-TSE direkt an der Kasse oder am PC stellen Sie bitte sicher, dass der dafür benötigte Treiber von Epson installiert ist. Dies gilt nicht für die Verwendung der Epson TSE in einem Epson-Server oder Epson-Printer.
+
 ## Pilotinstallation
 
 Für die Finalisierung der Epson TSE Imlementierung suchen wir Partner für eine Pilotinstallation.
```

Comment:
Maybe we should also remove that we are looking for pilot partner, as it is already finished now?